python/Makefile.am

```diff
@@ -0,0 +1,11 @@
+SWIG_SOURCES = analysis.i
+
+# TODO: Check libdivvun to remove call to all
```

Comment:
?

Reply:
`all` is a standard target of makefile, calling which doesn't seems necessary. Will fix this

Reply:
In regular `Makefile`, the standard target is the first one, which is by *convention* named `all`. But automake makes its own `all-am` thing based on bin_PROGRAMS, bin_SCRIPTS (which are given chmod +x), lib_LTLIBRARIES, EXTRA_DIST and such. If you want the file included when people make packages (`make dist`), it should go into the right such automake variable.